### PR TITLE
Add setup and run batch files

### PR DESCRIPTION
**Pull request by virtuallyaverage**
Add basic batch files to simplify running and setup.
The setup file should complete all steps in the instructions and point towards the next step. The only thing that really changed was having it setup the environment in the base directory, so .venv should probably be added to the gitignore. The environment is named jarves_venv and is activated in the run file. A shortcut can be created from the run.bat file so it can be put on the desktop or run on a scheduled task as your video showed.

This should not affect any current users, just make new installations smoother.

### CODE REVIEW

File: Assistant/VirtualAssistant.py
Comment (gianmarcoguarnier) [inline Assistant/VirtualAssistant.py:1]: If this change is meant to fix the early voice detection I'd rather keep the current version for now,  
```if is_voice and (time.perf_counter()-start_time)>self.MIN_RECORDING_TIME```
In this way the recording remains intact for whisper (0.5 seconds might not be a huge change)

**Review by gianmarcoguarnier — COMMENTED** [review]
The bat files seem a great addition and they'll definitely help the newcomers.

**Comment by virtuallyaverage** [comment]
glad to be helpful!

File: Assistant/VirtualAssistant.py
Comment (virtuallyaverage) [reply Assistant/VirtualAssistant.py:1]: I understand, I'll see about a solid solution later.

**Pull request closed without merging**